vip-helpers/class-vip-backup-user-role-cli.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/**
+ * VIP Backup User Roles CLI
+
+ * @package wp-cli
+ */
+class VIP_Backup_User_Role_CLI extends \WPCOM_VIP_CLI_Command {
+
+	/**
+	 * List Available Backups
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--format=<format>]
+	 * : The serialization format for the value. total_bytes displays the total size of matching options in bytes.
+	 * ---
+	 * default: table
+	 * options:
+	 *   - table
+	 *   - json
+	 *   - csv
+	 *   - count
+	 *   - yaml
+	 *
+	 * @subcommand list
+	 */
+	public function list_( $args, $assoc_args ) {
+
+		$backup_roles = get_option( 'vip_backup_user_roles', [] );
+		$backup_roles = array_reverse( $backup_roles, true );
+
+		$data = [];
+
+		foreach ( $backup_roles as $time => $roles ) {
+			$data[] = [
+				'time_key'  => $time,
+				'time_h'    => gmdate( 'Y-m-d H:i:s', $time ),
+				'num_roles' => count( $roles ),
```

Comment:
Would it be useful to show the role names here as well as a CSV string?

Reply:
I realize there's a `view` command as well, but I wonder if it would be more efficient if you can tell at a glance what each backup contains (rather than running `view` for each individually).

Reply:
simple enough, could add value 👍